src/test/users.test.js

```diff
@@ -0,0 +1,34 @@
+import chai from 'chai';
+import chaiHttp from 'chai-http';
+import faker from 'faker';
+import server from '../index';
+import {
+    status
+  } from '../utils/index';
+
+
+chai.use(chaiHttp);
+chai.should();
+
+// Sign Up User Testing
+describe('/POST new user', () => {
+  it('it should  POST a user', (done) => {
+    const user = {
+      email: faker.internet.email(),
+      password: faker.internet.password(),
+      first_name: faker.name.firstName(),
+      last_name: faker.name.lastName(),
+    };
+    chai.request(server)
```

Comment:
Hi @oyewoas, I don't think you need the faker package, you can just create a user like this 
```javascript
const user = {
  email: useremail@gmail.com,
   passsword: thepassword,
   first_name: John,
   last_name: Doe
}
```
 or did you use it for other reasons.

Reply:
I think it's ok to use it because it helps to generate random fake data rather than hardcoding it.